content/en/docs/tutorials/trust-policy.md

```diff
@@ -0,0 +1,263 @@
+---
+title: "Create a trust policy and verify artifacts"
+description: "How to create a trust policy and use it to verify artifacts"
+type: docs
+weight: 1
+---
+
+
+quick details on trust policy
```

Comment:
Should be removed?

Reply:
good call, i missed updating this content, fixed